app/lib/formatter.rb

```diff
@@ -69,6 +64,14 @@ def format_spoiler(status)
     html.html_safe # rubocop:disable Rails/OutputSafety
   end
 
+  def linkify(text)
+    html = encode_and_link_urls(text)
```

Comment:
does encode_and_link_urls sanitize correctly here? Or am I missing where the HTML-safety happens?

Reply:
Yes, it does that. This function has been extracted verbatim from simplified_format.

Reply:
✅